code/modules/vending/_vending.dm

```diff
@@ -636,8 +636,8 @@ GLOBAL_LIST_EMPTY(vending_products)
 		if(shock(user, 100))
 			return
 
-	if(tilted && !user.buckled)
-		to_chat(user, "<span class='notice'>You begin righting \the [src].")
+	if(tilted && !user.buckled && !isAI(user))
```

Comment:
does an ai also get shocked 3 lines above this?

Reply:
I assume since no one's ever ahelped that happening that it's handled somewhere else

Reply:
What about sentient diseases or people who are looking through (advanced) camera consoles?

Reply:
This check is bit weak yeah, check Adjacent, also why is this checking buckled, did you mean incapacitated.

Reply:
It's checking buckled because that's how I check to see if the person is pinned beneath the vending machine as one of the crits (since you can't be buckled to a vending machine otherwise)